primerpedia.js

```diff
@@ -206,7 +206,7 @@ window.onload = function () {
 	licenseIconElement = document.getElementById("license-icon");
 	infoIconElement = document.getElementById("info-icon");
 
-	var queryParam = getQueryVariable("search");
+	var queryParam = getQueryVariable("search").replace(/(\_|\+)/g, ' ');
```

Comment:
This will generate an error if `getQueryVariable()` returns `null`. Perhaps we should change it to return the empty string.

Reply:
Also, I don't think the underline needs escaping in regex. Nor we need to capture the match, so it could be simplified as `[_+]`.